docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst

```diff
@@ -2078,7 +2077,62 @@ Daml Script has been used for demonstrative purposes in this document, however
 usually the complexities of live upgrades comes with your workflows, not the data
 transformations themselves. You can use Daml Script (with Canton) to test some
 isolated simple cases, but for thorough tests of you system using SCU, you should
-prefer full workflow testing, as below.
+prefer full `Workflow Testing <#workflow-testing>`__.
+
+When testing your upgrades with Daml Script, we recommend you place your Daml Script tests
+in a separate package which depends on all versions of your business logic. This testing
+package should not be uploaded to the ledger if possible, as it depends on the ``daml-script-lts`` package.
+This package emits a warning on the participant when uploaded, as it serves no purpose on a participant,
+cannot be fully removed (as with any package), and may not be uploadable to the ledger in future versions (Daml 3).
```

Comment:
```suggestion
cannot be fully removed (as with any package), and may not be uploadable to the ledger in future versions (Daml 3). More information about this limitation :ref:`here <upgrade_dont_upload_daml_script>`.
```

Reply:
This reference will be there once https://github.com/digital-asset/docs.daml.com/pull/878 goes in

Reply:
This doesn't exist yet, the :ref: would error on build

Reply:
I suspect this PR will merge first, given it already has language team reviews. You may want to update this in your PR.
We'll see, race condition

Reply:
Yeah I figured - left it here not to forget

Reply:
@dylant-da This PR has merged, you'll need to make this addition on your side